domain/network/state/linklayer.go

```diff
@@ -7,6 +7,7 @@ import (
 	"context"
 
 	"github.com/canonical/sqlair"
+	"golang.org/x/exp/maps"
```

Comment:
Use the `maps` package in the standard library.

Reply:
It doesn't do the same thing, but I've used `transform` in any case.